### PR TITLE
Extra strings for localization

### PR DESCRIPTION
**Pull request by piotrpMSFT**
Strings the team feels may be needed before next localization opportunity.

/cc @krwq @MattGertz @blackdwarf

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.
@dotnet-bot test RHEL7.2 x64 Release Build please.

File: src/dotnet/LocalizableStrings.cs [hunk 1]
```diff
@@ -1,30 +1,180 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Microsoft.DotNet.Tools
+﻿namespace Microsoft.DotNet.Tools
 {
-    internal static class LocalizableStrings
+    internal class LocalizableStrings
```
Comment (nguerrera) [inline src/dotnet/LocalizableStrings.cs]: It doesn't really matter, but I'm curious if there was a reason  for dropping static here.

**Comment by piotrpMSFT** [comment]
statics are a pain to test

**Comment by nguerrera** [comment]
I understand the test concern in general, but this class has only const fields. What would you do with an instance?

**Comment by piotrpMSFT** [comment]
avoiding exceptions in the codebase.

**Pull request merged**